File: resources/androidstudio/app/src/main/java/com/nativephp/mobile/ui/nativerender/PressableRenderer.kt
package com.nativephp.mobile.ui.nativerender

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun RenderPressable(node: NativeUINode, modifier: Modifier) {
    // Click handling is done by buildModifier() (after all styling/sizing)
    Box(modifier = modifier) {
        for (child in node.children) {
            RenderNode(child)
        }
    }
}

File: resources/xcode/NativePHP/NativeRender/PressableRenderer.swift
import SwiftUI

struct RenderPressable: View {
    let node: NativeUINode

    var body: some View {
        ZStack {
            ForEach(node.children) { child in
                RenderNode(node: child)
            }
        }
        // Click handling is done by NodeModifier (after all styling/sizing)
    }
}
